apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Query_synthetic_id_ranges.sql

```diff
@@ -0,0 +1,102 @@
+-- Find the beneficiaries.bene_id range to avoid collisions with synthetic data.
+WITH ids AS (
+  SELECT
+    CAST(bene_id AS numeric) AS id
+  FROM beneficiaries
+  ORDER BY CAST(bene_id AS numeric)
+),
+groupings AS (
+  SELECT
+    ROW_NUMBER() OVER (ORDER BY id) - id AS grouping,
+    id
+  FROM ids
+)
+SELECT MIN(id) AS grouping_start,
+       MAX(id) AS grouping_end
+FROM groupings
+GROUP BY grouping
+ORDER BY MIN(id);
+
+-- Find  min claim Ids for all claims except Part D Events.
+-- This serves as an end range to avoid collisions with future synthetic data batches.
+select min(id) from (
+	select CAST(clm_id AS numeric) id 
+	from carrier_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from dme_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from hha_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from hospice_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from inpatient_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from outpatient_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	union
+	select CAST(clm_id AS numeric) id 
+	from snf_claims
+	where bene_id LIKE '-%' AND clm_id LIKE '-%'
+	) as ids;
+	
+-- Find  min Part D Event id range to avoid collisions with future synthetic data batches.
```

Comment:
same general comment as for clm_id. these two comments should be identical except for the name of the column